.github/workflows/test.yml

```diff
@@ -252,6 +257,8 @@ jobs:
       # coreutils: required by test-templates.sh for the "timeout" command
       # w3m :      required by test-templates.sh for port forwarding tests
       run: brew install qemu bash coreutils w3m
+    - name: "Run BATS integration tests"
+      run: make bats
```

Comment:
This should be probably executed on the Linux host for stability and performance

Reply:
I would eventually want to run it on all 3 platforms, but for now I've moved it to the  `test-misc.yaml` job on Linux.